### PR TITLE
add feature-file

### PR DESCRIPTION
**Pull request by tjkemper**
# Test PR

### CODE REVIEW

**Comment by octocat-jedi** [comment]
allo world!!!

**Comment by tjkemper** [comment]
hello from taylor

**Comment by octocat-jedi** [comment]
allo! again

**Comment by octocat-jedi** [comment]
comment

**Comment by octocat-jedi** [comment]
build

**Comment by octocat-jedi** [comment]
build

**Comment by octocat-jedi** [comment]
build

**Comment by octocat-jedi** [comment]
build

**Comment by octocat-jedi** [comment]
hello

**Comment by octocat-jedi** [comment]
allo

File: app/src/main/java/hello.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class hello {
```
Comment (octocat-jedi) [inline app/src/main/java/hello.java]: MINOR Move this left curly brace to the beginning of next line of code. [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
MINOR Remove this empty class, write its code or make it an "interface". [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS2094)
MINOR Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS00101)

**Comment by octocat-jedi** [comment]
SonarQube analysis reported 6 issues
* BLOCKER 1 blocker
* MINOR 5 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. BLOCKER [hello.java](https://github.com/octocat-jedi/boot-app/blob/6a88e8d053fda51eac4b88401fdb419b986e73df/app/src/main/java/hello.java): Add or update the header of this file. [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS1451)
1. MINOR [java](https://github.com/octocat-jedi/boot-app/blob/6a88e8d053fda51eac4b88401fdb419b986e73df/app/src/main/java): Add a 'package-info.java' file to document the 'java' package [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS1228)
1. MINOR [hello.java](https://github.com/octocat-jedi/boot-app/blob/6a88e8d053fda51eac4b88401fdb419b986e73df/app/src/main/java/hello.java): Move this file to a named package. [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS1220)

**Review by octocat-jedi — APPROVED** [review]

**Comment by octocat-jedi** [comment]
SonarQube analysis reported 6 issues
* BLOCKER 1 blocker
* MINOR 5 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. BLOCKER [hello.java](https://github.com/octocat-jedi/boot-app/blob/78d6229b655f26db62c0da3b9955febaf090ba4b/app/src/main/java/hello.java): Add or update the header of this file. [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS1451)
1. MINOR [java](https://github.com/octocat-jedi/boot-app/blob/78d6229b655f26db62c0da3b9955febaf090ba4b/app/src/main/java): Add a 'package-info.java' file to document the 'java' package [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS1228)
1. MINOR [hello.java](https://github.com/octocat-jedi/boot-app/blob/78d6229b655f26db62c0da3b9955febaf090ba4b/app/src/main/java/hello.java): Move this file to a named package. [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS1220)

**Comment by octocat-jedi** [comment]
allooo world

**Comment by octocat-jedi** [comment]
SonarQube analysis reported 6 issues
* BLOCKER 1 blocker
* MINOR 5 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. BLOCKER [hello.java](https://github.com/revature-training/boot-app/blob/8ede562bcb2d6fe3e7c9048693110464fc6cf45c/app/src/main/java/hello.java): Add or update the header of this file. [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS1451)
1. MINOR [java](https://github.com/revature-training/boot-app/blob/8ede562bcb2d6fe3e7c9048693110464fc6cf45c/app/src/main/java): Add a 'package-info.java' file to document the 'java' package [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS1228)
1. MINOR [hello.java](https://github.com/revature-training/boot-app/blob/8ede562bcb2d6fe3e7c9048693110464fc6cf45c/app/src/main/java/hello.java): Move this file to a named package. [rule](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS1220)

**Comment by octocat-jedi** [comment]
SonarQube analysis reported 6 issues
* BLOCKER 1 blocker
* MINOR 5 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. BLOCKER [hello.java](https://github.com/revature-training/boot-app/blob/4f9520ebbc6288df61eee3c2978eaa85ac47027a/app/src/main/java/hello.java): Add or update the header of this file. [rule](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3AS1451)
1. MINOR [java](https://github.com/revature-training/boot-app/blob/4f9520ebbc6288df61eee3c2978eaa85ac47027a/app/src/main/java): Add a 'package-info.java' file to document the 'java' package [rule](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3AS1228)
1. MINOR [hello.java](https://github.com/revature-training/boot-app/blob/4f9520ebbc6288df61eee3c2978eaa85ac47027a/app/src/main/java/hello.java): Move this file to a named package. [rule](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3AS1220)

**Pull request merged**